geode-docs/tools_modules/lucene_integration.html.md.erb

```diff
@@ -334,6 +334,7 @@ Lucene indexes created for all members.
 # <a id="LuceneRandC" class="no-quick-link"></a>Requirements and Caveats
 
 - Join queries between regions are not supported.
+- Executing queries is possible after all members are at the same version.
```

Comment:
This statement is not quite specific enough. I'm guessing that all members must be at the same minor version number (not necessarily the same patch version within the minor).  I'm also guessing that this is just for Lucene queries.

If I'm correct in my guesses, an attempt at better wording might be
> All cluster members must be running the same minor version in order to execute Lucene queries.

Reply:
Yes, you are right! Thanks!